gatsby/content/docs/online-events.mdx

```diff
@@ -5,7 +5,7 @@ section: Introduction
 slug: running-online-events
 ---
 
-Starting in 2020, team Matrix planned to start running regular meetup events, 
+Starting in 2020, Team Matrix planned to start running regular meetup events, 
```

Comment:
not sure I agree on this one - we don't usually refer to ourselves as "Team Matrix", I was just being figurative.

Reply:
The two options I see here are capitalising team or adding an article:
* I am with Team Matrix.
* I am with the team Matrix.